src/test/compile-fail/E0254.rs

```diff
@@ -11,7 +11,7 @@
 #![feature(collections)]
 
 extern crate collections;
-//~^ NOTE previous import of `collections` here
+//~^ NOTE previous import of the extern crate `collections` here
```

Comment:
I feel like this should just be the crate, there's no such thing as an "extern crate."